hooks/collect-metrics

```diff
@@ -9,64 +9,26 @@ from charms.layer.basic import activate_venv  # noqa: E402
 activate_venv()
 
 # Start the script as usual.
-import ssl  # noqa: E402
-import subprocess  # noqa: E402
-from urllib import request  # noqa: E402
-
-from firestealer import text_to_samples  # noqa: E402
+from firestealer import (
+    add_metrics,
+    retrieve_metrics,
+)  # noqa: E402
 import yaml  # noqa: E402
 
-
-# Map metrics names to samples to be collected.
-METRICS = {
-    'containers_in_flight': 'jujushell_containers_in_flight',
-    'errors_count': 'jujushell_errors_count',
-    'requests_count': 'jujushell_requests_count',
-    'requests_duration': 'jujushell_requests_duration_sum',
-    'requests_in_flight': 'jujushell_requests_in_flight',
-}
+from charms.layer import jujushell  # noqa: E402
```

Comment:
Why does lint choke on these imports? I'm surprised we need to noqa flag them.

Reply:
It wants all imports to be at the top of the file, while in here we have to do play with sys.path a bit before importing packages.